Assignment2/Assignment2_QNo1/SolAssignmentNo2_QNo1.rb

```diff
@@ -0,0 +1,32 @@
+# => 1. Write a class and define a class method which will return
+# => the object count of that class.
+# => Define the class
+class ObjectCnt
+  # => Define class variable initialize everytime new object is created
+  @@cnt_object = 0
+  def initialize
+    @@cnt_object += 1
+  end
+  # => Define the class method
+  def ObjectCnt.fn_cnt_object
+    @@cnt_object
+  end
+end
+
+# => Define user interface class
+class CallObjectCnt
+  def CallObjectCnt.user_access_interface
+    res = "y"
+    while (res == "y") do
+        # => Create object of the above class
+        puts "Creating object...."
+        obj = ObjectCnt.new
+        puts "Object count of the class is = #{ObjectCnt.fn_cnt_object}"
+        puts "Press y/n to continue/exit"
+        res = gets.chomp
+    end
+  end
+end
+
+# => Call the user interface method
+CallObjectCnt.user_access_interface
```

Comment:
@HrisikeshSinghaChowdhury End of file is missing.. You need to provide a new line in each file end otherwise this red icon will come in change list on github